portfolio/src/main/webapp/style.css

```diff
@@ -8,6 +8,14 @@
   margin-top: 20px;
 }
 
-img {
+/* img {
   border-radius: 50%;
+} */
+
+.container {
+  padding: 5%
+}
+
+#bg {
```

Comment:
It looks like you're using `bg` as a class so this will need to be `.bg` to take effect.

Alternatively, I have a hunch you're only ever going to apply this to the `body` element so changing the selector to `body` and not using a class or ID at all would also work.